Misc/NEWS.d/next/Core and Builtins/2022-07-30-14-10-27.gh-issue-95389.nSGEkG.rst

```diff
@@ -0,0 +1 @@
+Expose some of ``ETHERTYPE_*`` and ``ETH_P_ALL`` constants in :mod:`socket`. Patch by Noam Cohen.
```

Comment:
```suggestion
Expose :data:`~socket.ETH_P_ALL` and some of the
:ref:`ETHERTYPE_* constants <socket-ethernet-types>` in :mod:`socket`.
Patch by Noam Cohen.
```

* Clarify that `ETH_P_ALL` is a single constant, while `ETHERTYPE_*` is a group
* Link both
* Conform to an 80 column limit stated in the devguide